plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -43,7 +43,8 @@ import (
 )
 
 const (
-	PluginName = "PodSecurityPolicy"
+	PluginName           = "PodSecurityPolicy"
```

Comment:
we should also have the webhook as an example. My comment https://github.com/kubernetes/kubernetes/pull/58143/commits/e2b30326af2665f4e916e60571e57b5d06727972#r165015068 possibly conflicts with the webhook implementation. Only the webhook plugin (which is **one** plugin, or actually two for validating and mutating) knows about the registered webhooks. The admission chain does not.